src/test/java/org/pentaho/metadata/automodel/PhysicalTableImporterTest.java

```diff
@@ -66,10 +66,18 @@ public void testImportTableCanExcludeColumns() throws Exception {
       importTableDefinition( database, "aSchema", "aTable", "klingon", importStrategy );
     List<IPhysicalColumn> physicalColumns = physicalTable.getPhysicalColumns();
     assertEquals( 3, physicalColumns.size() );
-    assertContainsColumn( "quantityordered", physicalColumns );
-    assertContainsColumn( "productcode", physicalColumns );
-    assertContainsColumn( "status", physicalColumns );
 
+    // Depending on the class the formatting values are set or unset. If NULL formatting values were set on the
+    // property map, this would cause an XmiParsing Error. Testing that values that are NULL are not set in the
+    // property map.
```

Comment:
Sorry, only not I saw `assertNotNullPropertyValue` returns `true` **only** if the key is set.